### PR TITLE
Update data warehouse export UI

### PR DESCRIPTION
**Pull request by jtpalmer**
## Description

Adds a separate column for downloading exported data and adds the descriptions of all the fields that are exported for the selected realm.

Before:
data-export-before

After:
data-export-after

Also, refactor raw statistics configuration file use.

## Motivation and Context

https://app.asana.com/0/15230011293693/1137016263723284

## Tests performed

Altered automated tests and tested manually.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project as found in the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by jpwhite4** [comment]
Can you put a css style on the help text so that it matches the font face of the rest of the interface? It looks a bit jarring to me to have the help text in a different font.

File: classes/DataWarehouse/Data/RawStatisticsConfiguration.php [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+                $realm
+            ));
+        }
+
+        return array_map(
+            function ($field) {
+                // Replace hierarchy constants.
+                foreach (['name', 'documentation'] as $key) {
+                    $value = $field[$key];
+                    if (strpos($value, 'HIERARCHY_') === 0 && defined($value)) {
```
Comment (jpwhite4) [inline classes/DataWarehouse/Data/RawStatisticsConfiguration.php:144]: I thought that the XDMoDConfiguration class used backend code that translated the HEIRACHY variables automagically. Is this not the case on this code path?

**Comment by jtpalmer** [comment]
I've added the same CSS class as used by the Usage descriptions:
usage-tab-help-text

help-text-font-change

File: classes/DataWarehouse/Data/RawStatisticsConfiguration.php (hunk 1 above)
Comment (jtpalmer) [reply classes/DataWarehouse/Data/RawStatisticsConfiguration.php:144]: That is not the case on this code path.  (This isn't new code, it's just moved to a different file.)

I am not aware of any code path where that is the case, but if you know of one then please send me the details.

**Comment by jtpalmer** [comment]
@jpwhite4 please let me know if you have any other suggestions for the graphical design.  I couldn't think of much beside just putting everything in a list.  Also, do you think the heading ("{REALM} Realm Fields") is necessary or should otherwise be changed in content or style?

File: classes/DataWarehouse/Data/RawStatisticsConfiguration.php (hunk 1 above)
Comment (jtpalmer) [reply classes/DataWarehouse/Data/RawStatisticsConfiguration.php:144]: It looks like there is a feature in the group by refactor that does something like what you described, but it isn't implemented at the configuration class level.  It's part of `\Realm\Realm`.

Comment (jpwhite4) [reply classes/DataWarehouse/Data/RawStatisticsConfiguration.php:144]: Ok. I assumed that the variable substitution stuff was implemented lower down the stack that in actually is.

**Comment by jpwhite4** [comment]
The title "{REALM} realm fields" seems OK to me.

**Pull request merged**